release/src/router/rom/webs_scripts/merlin_webs_update.sh

```diff
@@ -93,11 +93,11 @@ if [ "$webs_state_flag" -eq "1" ]; then
 	if [ "$forsq" == "1" ]; then
 		echo "---- download SQ release note $fwsite/test/$releasenote_file0_US ----" >> /tmp/webs_upgrade.log
 		/usr/sbin/wget $wget_options $fwsite/test/$releasenote_file0_US -O $releasenote_path0
-		echo "---- $fwsite/test/$releasenote_file0 ----" >> /tmp/webs_upgrade.log
+		echo "---- $fwsite/test/$releasenote_file0_US ----" >> /tmp/webs_upgrade.log
```

Comment:
$releasenote_file0 is not a valid variable
It should be $releasenote_file0_US

![image](https://github.com/gnuton/asuswrt-merlin.ng/assets/1971404/af6bb811-1718-4905-a03d-44a02963e1eb)

Reply:
> 
> $releasenote_file0 is not a valid variable It should be $releasenote_file0_US
> 
Nice catch!!

Reply:
Cleaned up to match RMerlins commit: https://github.com/RMerl/asuswrt-merlin.ng/commit/90a9cc3a53e0415056949e0eec6318bb28244c9a

Resolving review note.